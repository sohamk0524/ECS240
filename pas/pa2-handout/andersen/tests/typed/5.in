c Test case with address-of and multiple derefs on LHS and RHS
p 14 13
t 1 5
t 2 5
t 3 5
t 4 4
t 5 4
t 6 4
t 7 3
t 8 3
t 9 3
t 10 2
t 11 1
t 12 1
t 13 0
t 14 0
c layer 5
s 0 1 -1 4
s 0 2 -1 5
s 0 3 -1 6
c layer 4
s 1 1 -1 7
s 1 2 -1 8
s 1 3 -1 9
c layer 3
s 2 3 -1 10
s 2 2 1 6
s 1 4 2 2
c layer 2
s 3 1 -1 11
s 3 2 -1 12
c layer 1
s 4 3 -1 13
s 0 12 -1 14
