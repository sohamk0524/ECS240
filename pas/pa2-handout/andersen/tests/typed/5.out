pt 10 11
pt 10 12
pt 11 13
pt 12 13
pt 12 14
pt 1 4
pt 2 5
pt 3 6
pt 4 7
pt 5 8
pt 6 9
pt 7 10
pt 8 10
pt 9 10
